src/components/manage-communications.js

```diff
@@ -0,0 +1,308 @@
+import React from "react"
+import { connect } from "react-redux"
+import { startCase, toLower } from "lodash"
+import { Checkbox, Segment, Accordion, Icon, Button, Message } from "semantic-ui-react"
+
+import CustomBreadcrumb from "core/common/src/components/custom-breadcrumb"
+import { appDetails, Loading } from "formula_one"
+import { getSubscriptionCategoryList, submitSubscription } from "../actions"
+import Sublist from "./manage-communications/sublist"
+
+import "../css/communications.css"
+import { appBaseURL } from "../urls"
+import { getTheme } from "../../../../formula_one/src/utils"
```

Comment:
It can be imported directly from the "formula_one" no need to give the full_url, you may even put it with the 7th line import